tests/store_test.go

```diff
@@ -40,7 +40,7 @@ type StoreSuite struct {
 func (s *StoreSuite) TestStoreNew() {
 	doc := NewStoreWithConstructFixture("foo")
 	s.False(doc.IsPersisted())
-	s.False(doc.ID.IsEmpty())
+	s.True(doc.ID.IsEmpty())
```

Comment:
I think that in this PR, the old test assertion should be maintained, implementing the "ID generation" in the fixture `newStoreWithConstructFixture(f string) *StoreWithConstructFixture` method, to ensure that the tested behavior is maintained;